src/sentry/static/sentry/app/components/feature.jsx

```diff
@@ -42,6 +50,16 @@ class Feature extends React.Component {
      */
     isSuperuser: PropTypes.bool,
 
+    /**
+     * Show a default message when user does not have feature flag enabled
+     */
+    showNoFeatureMessage: PropTypes.bool,
+
+    /**
+     * Custom renderer for "no feature" message
+     */
+    renderNoFeatureMessage: PropTypes.func,
```

Comment:
How about `renderNoFeatureMessage` takes `PropTypes.oneOf([PropTypes.func, PropTypes.bool])`, then you can either turn it on with `true`, or specify a custom render function. Or were you anticipating maybe specifying a custom render function, but still wanting to be able to turn it off.

Reply:
https://github.com/getsentry/sentry/pull/9190/commits/7f34fbded22a4488c785b0629e26ebfb337fbbc9